contracts/src/PriceFeedImporter.sol

```diff
@@ -84,15 +109,22 @@ contract PriceFeedImporter is EventImporter {
         _onlyMoreRecentEvents(eventInfo)
     {
         // Update the latest answer.
-        currentAnswer = int256(uint256(eventInfo.log.topics[1]));
-        roundID = uint80(uint256(eventInfo.log.topics[2]));
-        updatedAt = uint256(bytes32(eventInfo.log.data));
+        uint80 roundID = uint80(uint256(eventInfo.log.topics[2]));
+        if (roundID <= latestRoundID && latestRoundID != 0) {
+            revert("roundID should be monotonically increasing");
+        }
```

Comment:
Just need to make sure that prices imported are not duplicated but they could be imported out of order while the view function would not return the last answer if the previous ones are missing.

Reply:
Opened https://github.com/ava-labs/event-importer-poc/issues/78 to unblock this.